torch/onnx/symbolic_opset9.py

```diff
@@ -1234,7 +1234,15 @@ def batch_norm(g, input, weight, bias, running_mean, running_var, training, mome
         bias_value = torch.tensor([0.] * input_sizes[1]).type(
             'torch.' + input.type().scalarType() + 'Tensor')
         bias = g.op("Constant", value_t=bias_value)
-
+    # If track_running_stats is set to False batch statistics are instead used during evaluation time    
+    if running_mean is None or sym_help._is_none(running_mean) or running_var is None or sym_help._is_none(running_var):
+        assert len(input_sizes) > 1
+        reshape_in = g.op("Reshape", input, 
+                          g.op("Constant", value_t=torch.tensor([input_sizes[0], input_sizes[1], -1], dtype=torch.int64)))
+        trans_in = g.op('Transpose', reshape_in, perm_i=[0, 2, 1])
+        running_var, running_mean = _var_mean(g, trans_in, 
+                                              g.op("Constant", value_t=torch.tensor([0, 1], dtype=torch.int64)), 
```

Comment:
I wonder if previous transpose is still necessary if we are able to pass dim to reduce over here `[0, 1]`?

Reply:
Based on current implementation of var_mean, values of dim have to be sequential and start with zero. I'm not sure what is causing this behavior but these formats listed below are causing an incompatible dimension for the Sub node. 
-> dim[1, 2]
-> dim[0, 2]
-> dim[2, 0] etc

Reply:
Will revisit the issue with var_mean later. There is also an item on adding support for opset 13 updates. I'm linking the discovery here to that issue.